anomalib/pre_processing/pre_process.py

```diff
@@ -8,16 +8,56 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import logging
+import warnings
 from typing import Optional, Tuple, Union
 
 import albumentations as A
 from albumentations.pytorch import ToTensorV2
 
-from anomalib.data.utils import get_image_height_and_width
-
 logger = logging.getLogger(__name__)
 
 
+def get_image_height_and_width(image_size: Optional[Union[int, Tuple]] = None) -> Tuple[Optional[int], Optional[int]]:
```

Comment:
Once we remove `PreProcessor`, I think we would remove the `pre_process.py` file as well. Do you think if this is the best place for this function?

Reply:
This is the legacy implementation of this function, which is needed by the `PreProcessor` class. So this is just for backward compatibility, and we can safely remove it once we remove `PreProcessor`. The new function that's actually being used throughout the code is in `anomalib.data.utils.image.get_image_width_and_height`.

Reply:
oh, I see. Thanks for the clarification